pkg/internal/apis/certmanager/validation/issuer.go

```diff
@@ -35,7 +35,19 @@ import (
 
 func ValidateIssuer(obj runtime.Object) field.ErrorList {
 	iss := obj.(*v1alpha1.Issuer)
-	allErrs := ValidateIssuerSpec(&iss.Spec, field.NewPath("spec"))
+	fldPath := field.NewPath("spec")
```

Comment:
I'm not particularly in love with putting the validation here, but it was the least intrusive way I could think of.  ValidateIssuer does strictly validate Issuer types, but ValidateIssuerSpec will tolerate specs from either an Issuer or a ClusterIssuer.